### ISSUE TITLE
Kernel dies when trying to import shap latest 2 versions

### ISSUE THREAD

**Issue by guyts** [shap/shap#2046]
When trying to import `shap` into my notebook (working either with Jupyter Lab or VS code), my kernel keeps on dying due to `shap` alone (see example screengrab here).

Error message reads: `The kernel for Untitled.ipynb appears to have died. It will restart automatically.`.

This was true for both latest version (0.39) and previous version (0.38.1); also made sure both were installed with `sudo` just to verify it's not due to some issue with installation permissions.

When doing something similar with versions 0.36 and 0.37, there are no such issues.

Find below other libraries used in this project for reference.

Using `pip 21.1.2` and Python 3.9 on a Mac (M1, but running on Rosetta terminal).

Does anyone have any lead to what might be the possible issue ? 

```
xgboost==1.4.2
joblib==1.0.1
boto3==1.17.89
seaborn==0.11.1
matplotlib==3.4.2
shap==0.38.1
pandas==1.2.4
numpy==1.19.5
tqdm==4.61.0
lightgbm==3.2.1
geopandas==0.9.0
requests==2.25.1
eli5==0.11.0
click==8.0.1
interpret==0.2.4
scikit_learn==0.24.2
```

**Comment by denis-gordeev** [comment]
I have also this problem. It might be irrelevant but Tensorflow >= 2.3.0 also has this problem for processors without AVX support.

**Comment by drsanchikagupta01** [comment]
I am also experiencing the same problem. It's in the M1 processor only. Any fixes?

**Comment by tommasodiotalevi** [comment]
I also have this issue, I cannot install shap from version 0.36.0. I receive the error
`Segmentation Fault: 11`
I have an old macbook, with the Intel processor (not M1)

**Comment by Veghit** [comment]
+1

**Comment by pascalamadeus** [comment]
I have the same problem using version 0.42.1 in jupyter notebook
